eng/pipelines/notifications.yml

```diff
@@ -30,6 +30,8 @@ stages:
           PathPrefix: 'cpp'
         Go:
           PathPrefix: 'go'
+        AzDev:
+          PathPrefix: 'azure-accelerators'
```

Comment:
Is this how we have named the pipelines? We may need to shorten this to something like azd because I know the team names can get a little long.

Reply:
Why not just `AzDev`?

Reply:
`PathPrefix` is the `azure-accelerators` part of `internal\azure-accelerators\azure-dev - etc.etc.` ... I'm going to change it to `azure-dev` to match the pipeline names. It'll match the first part of the branded name ([Azure Developer CLI](https://docs.microsoft.com/en-us/azure/developer/azure-developer-cli/overview?tabs=nodejs))